src/client/Cargo.toml

```diff
@@ -4,6 +4,8 @@ version = "0.3.6"
 edition = "2018"
 license = "Apache-2.0"
 authors = ["Fluvio Contributors <team@fluvio.io>"]
+categories = ["database"]
+keywords = ["streaming", "stream", "queue", "kafka"]
```

Comment:
remove kafka

Reply:
Yeah, I was on the fence about that. I went with it because maybe people searching for kafka would find us. Anyway, I've removed it.